noetic/distribution.yaml

```diff
@@ -3389,6 +3389,21 @@ repositories:
       url: https://github.com/ros-drivers/urg_c.git
       version: master
     status: maintained
+  urg_node:
+    doc:
+      type: git
+      url: https://github.com/ros-drivers/urg_node.git
+      version: kinetic-devel
```

Comment:
The Kinetic and Melodic versions of this package are both on the "indigo-devel" branch.  The noetic one here is being setup for "kinetic-devel", which is fine.  But it seems like if "kinetic-devel" supports everything back to Kinetic, then all of Kinetic, Melodic, and Noetic should use that same branch.  Thoughts?

Reply:
I agree, the only reason for the kinetic-devel was to bump the CMakeLists version.

Reply:
Do you want me to update the tracks now?

Reply:
Yeah, you can go ahead and do that.  Typically it would be separate PRs, but right now all of Kinetic, Melodic, and Noetic are *not* in freeze.  If we get it in today, we can do all of them at once.

Reply:
Whenever I run `bloom-release --edit-track -r kinetic -t kinetic urg_node` the information is pre-filled with kinetic-devel.  However, the PR that is generated doesn't change indigo-devel to kinetic-devel.  Any idea why not?

Reply:
Hm, sorry, no idea here.  In theory bloom can do this, but I typically do it manually.